pkg/operator/cluster.go

```diff
@@ -133,28 +129,29 @@ func (c *Cluster) createClientAccess(clusterInfo *cephmon.ClusterInfo) error {
 	}
 
 	// store the secret for the rbd user in the default namespace
```

Comment:
This is done to make it easier for users to get a quick start.  The RBD volume plugin expects a secret on the same namespace as the app with a valid ceph token.